server/graphql/core/src/simple_generic_errors.rs

```diff
@@ -340,3 +340,11 @@ impl MasterListNotFoundForThisStore {
         "Master list not found (might not be visible to this store)"
     }
 }
+
+pub struct FailedTranslation;
+#[Object]
+impl FailedTranslation {
+    pub async fn description(&self) -> &'static str {
+        "Translation failed."
+    }
+}
```

Comment:
I have added this as a generic error for future use case of translating any json.